---
layout: people
title: Keith R. Yamamoto, PhD
permalink: /yamamoto/

---
![yamamoto pic](../img/kry.jpg)


keith.yamamoto@ucsf.edu

 

Keith Yamamoto is Vice Chancellor for Research at UCSF, Executive Vice Dean of the School of Medicine, and Professor of Cellular & Molecular Pharmacology. He has been a member of the faculty at UCSF since 1976. For Dr. Yamamoto’s complete CV, click here.

  

**Research**

 

Understanding gene-, cell- and physiology-specific transcriptional regulation in metazoans. Metazoan organismal development, maintenance of metabolic homeostasis, and response to physiological challenges such as stress, trauma and infection, are strongly dependent on temporal and spatial regulation of mRNA accumulation. The production of mRNA is itself complex, involving machinery and mechanisms for initiation, elongation, splicing, cleavage and polyadenylation, transport and degradation; indeed, most of these processes are tightly coupled in vivo. In principle, any of these steps could be modulated by the extensive array of metazoan transcriptional regulatory factors, and it is clear that mRNA expression is exquisitely specific. Understanding this specificity is important because it is the basis for a broad array of biological processes, and because defects in specificity are at the heart of many developmental anomalies and diseases.   

Our lab studies members of the nuclear receptor (NR) superfamily, an essential class of metazoan transcriptional regulatory factors. Upon binding cognate hormones, we showed that the glucocoticoid receptor (GR), the founding member of the superfamily, binds to specific genomic sites termed glucocorticoid response elements (GREs), and activates or represses transcription of nearby target genes; regulation, it appeared, would be elegant and simple. Since those discoveries in the early 1980’s, we and others have come to appreciate that metazoan transcriptional regulation is indeed elegant, but far from simple. We now know, for example, that GREs are composite elements, comprised of specific GR binding sequences linked to gene-specific arrays of sequence motifs recognized by non-GR regulatory factors, and that response elements are nucleation centers for the dynamic assembly and disassembly of multifactor regulatory complexes containing gene-, cell- and physiologic state-specific combinations of roughly 102 different genome binding regulatory factors and coregulatory factors associated through protein:protein interactions.
    

Two additional levels of combinatorial complexity, observed with GR and many other regulators, should be noted. First, “crosstalk”, in which unrelated regulatory factors affect each other’s activities, demonstrates that metazoan regulation is specified by nonlinear networks. Second, coregulators display a spectrum of different activities (recruitment of general transcription factors, addition or removal of histone marks, posttranslational modifications of nonhistone proteins, chromatin remodeling), implying that regulation from a given response element is likely comprised of multiple “layers” of different mechanisms.

    

Mechanisms of combinatorial specificity and relationship to physiological outcome.  Although the concept of combinatorial regulation is firmly established, perhaps the primary barrier to progress is that we understand little about the mechanisms of specificity, i.e., determinants that enable broadly expressed factors to assemble into multiple regulatory complexes with distinct compositions, geometries and functions. We know that it is insufficient to invoke cell-specific regulatory factors to account for such specificity, as this merely begs the question of the specificity mechanisms that produce such factors; moreover, factor expression is generally rather broad.
    

Consequences of this barrier are that we cannot recognize or predict functional GREs, nor do we understand drivers of combinatoriality and their implications for layering and overall regulation. Based on our work and that of others, we proposed that specificity is conferred by signaling, that signaling imposes allosteric conformational changes on targets including transcriptional regulatory factors such as GR, that key factors like GR integrate information from multiple signals, and that the resultant conformations produce patterns of functional surfaces that nucleate assembly of particular regulatory complexes.
     

A second barrier to progress is the open question of the relationship of regulatory mechanisms to physiological outcomes.  There is currently no direct evidence linking metazoan mechanisms to phenotypes, and complex networks could complicate detection of such relationships. However, in prokaryotes, where regulatory circuits are simple, such relationships are plainly evident. For example, the “response elements” that control operons for catabolism of various sugars all display a characteristic structure and mechanism, with a sugar-specific repressor binding site overlapping the promoter, and a cyclic AMP receptor binding site just upstream of the repressor site, which is occupied and activates transcription in the absence but not the presence of glucose. In contrast, the amino acid biosynthetic operons are regulated by an attenuation mechanism in which ribosomes stall during translation of a 5’ leader peptide only if the cognate amino acid is unavailable, thus preventing formation of an RNA stem-loop transcription termination signal.                                                                                               

Consequences of this barrier are that we lack mechanistic links to gene networks. Our approach to identifying such links is to interrogate discrete networks defined by “Causative Primary Regulated Genes" (CPRGs), i.e., genes linked to a GR-occupied GREs and essential in that context for a glucocorticoid-regulated action or phenotype, using GR as a probe of different signaling contexts that affect a given phenotype. Even if we fail to demonstrate that genes contributing different steps in a physiologic process share molecular properties of regulatory mechanisms that are distinct from those for other physiologic processes, our work will identify clinically important GR-regulated genes, and gain important insights into the organization and function of regulatory networks.

     

Relating gene networks, regulatory mechanisms and physiology; understanding protein allostery. In classical studies, Jacob and Monod in 1961 established the paradigm that transcription can be controlled by regulatory factors bound to promoter proximal genomic sites, but did not account for differential regulation in different cells types. Britten and Davidson in 1969 proposed that combinatorial principles could generate different regulatory complexes under different conditions, but did not explain the determinants of specificity for different combinations of common factors. We know now that metazoan transcriptional regulation operates in complex nonlinear networks, fed by multiple signaling inputs. We propose to approach this daunting complexity by anchoring our focus on genes directly regulated by GR, using as a probe one of the best-studied metazoan transcriptional regulators, a physiologically essential factor known to integrate multiple signals. From those anchor points, signaling inputs may be examined singly and in combination. Thus, we seek in our research to: [a] define primary GR networks; [b] show how combinatorial specificity produces allosteric changes in GR conformation, nucleating assembly of distinct regulatory complexes; [c] determine relationships between molecular regulatory mechanisms and physiologic outputs; [d] identify CPRGs that produce clinically important outcomes; and [e] establish CPRG GREs as control points that could in principle be targeted for therapeutic intervention.




**Teaching**

Over the course of Dr. Yamamoto’s career, he has been deeply involved in teaching, both at the professional and graduate school levels, and his efforts have been recognized both at the national (Dreyfus Teacher-Scholar Award) and the local (UCSF Distinguished Teaching Award) levels. 

Dr. Yamamoto co-developed (with Christine Guthrie) in 1977 the core graduate molecular biology course, Biochem 201A, Biological Regulatory Mechanisms, directed that course for over twenty years, and has taught in the course for 37 consecutive years. It is regarded as the most rigorous of UCSF’s graduate courses, yet is also well-liked and respected by the students. He also co-developed (with Henry Bourne) a cell signaling graduate course, BMS 227, and taught in that course for three years (1995-1997). In the UCSF professional schools, Dr. Yamamoto has presented lectures on topics in biochemistry, molecular biology and pharmacology to the medical, pharmacy, dentistry and nursing students, and has led small groups in the Prologue block of the current medical curriculum, which focuses on basic concepts of drug action, pharmacology, pharmacokinetics and pharmacodynamics.

Dr. Yamamoto helped to originate (and promoted at the national level) teaching on the ethics and practice of science, and teaches every year in the course on these topics (BMS 214/Neuroscience 214). He also present lectures on grant writing annually to graduate students (CCB170) and to MSTP students.

 
 
